modules/host/persist.go

```diff
@@ -172,26 +171,6 @@ func (h *Host) load() error {
 		return err
 	}
 
-	// Get the contract count by observing all of the incomplete storage
-	// obligations in the database.
-	err = h.db.View(func(tx *bolt.Tx) error {
```

Comment:
The database scan was added because other accounting in the host is incorrect. I'd much prefer to keep this, and set h.financialMetrics.ContractCount to 0 at start.

The scan I think we'll be adding other features to. The whole host is a mess right now and I think we need to inject some sanity. Recalculating values at startup is going to be more reliable than keeping some state around and hoping it never corrupts.

At least temporarily anyway.